docs/api/renderer.lua

```diff
@@ -41,10 +43,22 @@ function renderer.font.load(path, size, options) end
 ---
 ---Clones a font object into a new one.
 ---
+---@param font renderer.font
 ---@param size? number Optional new size for cloned font.
 ---
 ---@return renderer.font
-function renderer.font:copy(size) end
+function renderer.font.copy(font, size) end
```

Comment:
What's the reason for this change?

Reply:
did that so it is in line with load and group, should provide same autocompletion as before

Reply:
Should we also change `renderer.font:set_size` then?

Reply:
Now that you mention I left that in place because I don't know if it is going to be re-implemented or I should just remove it.

Reply:
Oh yeah, you're right. I think this is a question for @adamharrison.

Reply:
Still, the same original question applies to `renderer.font:set_tab_size`, `renderer.font:get_width`, `renderer.font:get_height` and `renderer.font:get_size`.

Reply:
should I change them all?

Reply:
I think I will just move copy below group and leave it as before

Reply:
ok, don